app/components/Dashboard/AssetBalancesPanel/index.js

```diff
@@ -29,6 +32,18 @@ const mapPricesDataToProps = ({ NEO, GAS }) => ({
   gasPrice: GAS
 })
 
+const mapPriceChangeDataToProps = (prices, props) => {
```

Comment:
is props being used? can it be dropped?
also, could you describe the shape of `prices`?

finally, I think its good to return to type too

```
const mapPriceChangeDataToProps = (prices, props): { neoPriceChange: BigNumber, gasPriceChange: BigNumber }  => {
}
```

Reply:
Props isn't being used so I can remove that, but I don't think return types will do us any good. There are dozens of these mapXDataToProps functions in the code and not one of them has a return type. I think this is because it's just injecting the resulting object into the properties.